community/socklog/socklog.post-install

```diff
@@ -11,12 +11,11 @@ if [ ! -d /etc/sv/socklog-unix ]; then
 fi
 
 if [ ! -f /etc/sv/socklog-unix/run ]; then
-cat > /etc/sv/socklog-unix/run <<EOF
-#!/bin/sh
-/etc/init.d/syslog stop &>/dev/null
-exec 2>&1
-exec chpst -Unobody socklog unix /dev/log
-EOF
+	cat > /etc/sv/socklog-unix/run <<EOF
```

Comment:
No, this doesn’t remove the leading indentation. It should be `<<-EOF`, it’s not a typo, [see](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_07_04). ;)